src/components/Book.js

```diff
@@ -1,31 +1,54 @@
-import React from 'react';
+/* eslint-disable */
+
```

Comment:
- [ ]  For best practice, kindly make sure you fix your linters errors instead of dasabling them for the enitre file.

Reply:
@tresorsawasawa Thanks for the review. I know how important eslint is, but I disabled it because the linter errors only appeared on GitHub and not on my vscode. I didn't know how else to fix it on GitHub since I had already fixed it in my local repo.(That is, my app ran locally without errors of any sort)

Reply:
Hi @blase147 

Sorry to answer you later. It's not permitted to disable linters for the entire file kindly try fixing it and also ask for help if you get stuck.